postgres/datadog_checks/postgres/data/conf.yaml.example

```diff
@@ -55,6 +59,45 @@ instances:
 #         pending_events: [postgresql.londiste_pending_events, GAUGE]
 #      query: SELECT consumer_name, %s from pgq.get_consumer_info() where consumer_name !~ 'watermark$';
 #      relation: false
+#
+#
+# Providing custom queries is also supported. Each query must have 3 fields:
+#
+# 1. metric_prefix - This is what each metric will start with.
+# 2. query - This is the SQL to execute. It can be a simple statement or a
+#            multi-line script. Only the first row of the result is read.
+# 3. columns - This is a list representing each column, ordered sequentially
+#              from left to right. There are 2 required pieces of data:
+#                a. name - This is the suffix to append to the metric_prefix
+#                          in order to form the full metric name. If `type` is
+#                          `tag`, this column will instead be considered a tag
+#                          and will be applied to every metric collected by
+#                          this particular query.
+#                b. type - This is the submission method (gauge, count, etc.).
+# 4. tags (optional) - A list of tags to apply to each metric.
+#
+#    custom_queries:
+#      - metric_prefix: postgresql
+#        query: |  # Use the pipe if you require a multi-line script.
+#          SELECT consumer_name,
+#                 GREATEST(0, EXTRACT(EPOCH FROM lag)) as lag,
+#                 GREATEST(0, EXTRACT(EPOCH FROM lag)) as last_seen,
+#                 pending_events
+#          FROM pgq.get_consumer_info()
+#          WHERE consumer_name !~ 'watermark$';
+#        columns:
+#          # Put this for any column you wish to skip:
+#          # - {}
```

Comment:
can we make this a bit clearer? Like with a real example

Reply:
Also I'd rephrase to something like

> Columns without a name are ignored, put this for any column you wish to skip:

Reply:
I've thought about this some more and I'm not sure if I can come up with a real world example. The use-case for skipping columns would be when the query is a `SELECT * ...` and the user does not want to specify the columns individually.

Reply:
The new wording is ok for me.

Reply:
Can we remove the ability to skip specific columns? Like @rishabh mentioned, the only case here seems to be if you are doing a `SELECT *`, but in that case you don't know exactly what you're getting back and its not trivial to tell what you should list in the column section anyway? 

If you are doing a `SELECT`on specific columns, then you just wouldn't select columns that you would ignore later anyway?

Perhaps there is a use case I'm missing?

Reply:
I feel the same way, but it seems like there _could_ be a use case for this, e.g. the user wants to quickly write a config that involves copy-pasting queries.

But in any case, I think we should keep it in to maintain feature parity with Oracle DB's virtually identical `custom_queries` config option. Or we could remove the ability to skip columns in the Oracle DB check and not include it for Postgres.